alembic/templates/async/alembic.ini.mako

```diff
@@ -72,6 +72,12 @@ sqlalchemy.url = driver://user:pass@localhost/dbname
 # black.entrypoint = black
 # black.options = -l 79 REVISION_SCRIPT_FILENAME
 
+# format using "ruff" - use the exec runner, execute a binary
```

Comment:
**mike bayer** (zzzeek) wrote:

"ruff" is a linter, I see "--fix" attempts to fix things, OK we should rephrase this as "lint with attempts to fix using ruff", it doesnt seem like this is exactly a "formatter" ...thoughts?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4788